OSDOCS-5334 Adding namespace option to oc adm  must-gather (#56342)
openshift/openshift-docs: modules/about-must-gather.adoc

...
----
// todo: table or ref module listing available images?

> Optionally, you can run the `oc adm must-gather` command in a specific namespace by using the `--run-namespace` option.

Reviewer: ```suggestion
Optionally, you can use the `--run-namespace` option to specify a namespace to run the `must-gather` tool in.
```

Author: Thanks!

Reviewer 2: Are you going to implement Andrea's feedback?

Author: We talked about it and she said it was fine the way it was. Would you like me to change it?

Reviewer: Yep we talked about it and I think this is good the way it is. Thanks for checking!
